tox.ini

```diff
@@ -1,10 +1,21 @@
 [tox]
 envlist =
-    py26,py27
+    py27,
+    py35,
+    py36,
 
 [testenv]
 deps =
     zope.testrunner
     .[test]
 commands =
     zope-testrunner --test-path=src {posargs:-pvc}
+
+[testenv:coverage]
+usedevelop = true
+deps =
+    {[testenv]deps}
+    coverage
+commands =
+    coverage run -m zope.testrunner --test-path=src {posargs:-pvc}
+    coverage report -m
```

Comment:
A `--fail-under=XX` might be helpful. Even if XX is not 100% yet it at least captures the current state and prevents regressions.